src/Utils/Helpers.ts

```diff
@@ -17,9 +19,47 @@ export interface CommandArgs {
   args: string[];
 }
 
+export interface ClusterUrls {
+  api?: string;
+  bus?: string;
+}
+
 export class Helpers {
   private static readonly META_CHARS_REGEXP = /([()\][%!^"`<>&|;, *?])/g;
 
+  public static getClusterUrls(args: Arguments): ClusterUrls {
+    let bus: string;
+    let api: string;
+
+    if (args.cluster) {
+      if (args.api || args.bus) {
+        throw new Error(
+          '--api and --bus options should not be used with --cluster option'
+        );
+      }
+      let host = args.cluster;
+
+      try {
+        host = { host } = new URL(args.cluster as string);
```

Comment:
It's a buggy line, `host` will be equal instance of URL  
```suggestion
        ({ host } = new URL(args.cluster as string));
```